### ISSUE TITLE
piControl aligned with Historical data

### ISSUE THREAD

**Issue by ledm** [ESMValGroup/ESMValCore#342]
I want to de-trend a historical simulation with the simultaneous patch of the piControl simulation. 

The detrending part is fine - I'll do that in the diagnostic - but is there a way to automate extracting the relevant years from a piControl run?

**Comment by zklaus** [comment]
Not quite, I don't think. But we took care to make sure that the relevant attributes remain correct, that is you have to look for `branch_time_in_parent` and `parent_time_units` to identify the starting point of the experiment. You can combine this information with `branch_time_in_child` to understand which are the overlapping parts.

**Comment by ledm** [comment]
Cheers @valeriupredoi, @zklaus. 

I don't think detrend preprocessor will help in my case. We need to detrend the historical run using the piControl data.

**Comment by zklaus** [comment]
I agree that this preprocessor will not help you here.
